src/azure-cli/azure/cli/command_modules/synapse/commands.py

```diff
@@ -119,8 +149,57 @@ def get_custom_sdk(custom_module, client_factory):
         g.custom_command('update', 'update_sql_pool')
         g.command('pause', 'pause')
         g.command('resume', 'resume')
+        g.custom_command('restore', 'restore_sql_pool')
+        g.custom_command('show-connection-string', 'sql_pool_show_connection_string')
         g.wait_command('wait')
 
+    # Management Plane Commands --SqlPool list-deleted
+    with self.command_group('synapse sql pool', command_type=synapse_restorable_dropped_sqlpools_sdk,
+                            client_factory=cf_synapse_client_restorable_dropped_sqlpools_factory) as g:
+        g.command('list-deleted', 'list_by_workspace')
+
+    #  Management Plane Commands --SqlPool Classification
+    with self.command_group('synapse sql pool classification', command_type=synapse_sqlpool_sensitivity_labels_sdk,
+                            custom_command_type=get_custom_sdk('sqlpoolsensitivitylabel',
+                                                               cf_synapse_client_sqlpool_sensitivity_labels_factory),
+                            client_factory=cf_synapse_client_sqlpool_sensitivity_labels_factory) as g:
+        g.custom_show_command('show', 'sqlpool_sensitivity_label_show')
+        g.command('list', 'list_current')
+        g.custom_command('create', 'sqlpool_sensitivity_label_create')
```

Comment:
That is the new added cmdlet? 
@idear1203  Can we use set to combine `create` and `update` ?

Reply:
just for FYI, sql db only has `update`

---
In reply to: [533201557](https://github.com/Azure/azure-cli/pull/16095#discussion_r533201557) [](ancestors = 533201557)

Reply:
I don't know the answer. @jsntcy , any thoughts on this?

---
In reply to: [533366840](https://github.com/Azure/azure-cli/pull/16095#discussion_r533366840) [](ancestors = 533366840,533201557)